dcrwallet.go

```diff
@@ -309,7 +305,14 @@ func rpcClientConnectLoop(legacyRPCServer *legacyrpc.Server, loader *wallet.Load
 			if legacyRPCServer != nil {
 				legacyRPCServer.SetChainServer(chainClient)
 			}
-			c <- chainClient
+			if cfg.EnableTicketBuyer {
+				loader.SetChainClient(chainClient.Client)
+				err := loader.StartTicketPurchase(nil, &cfg.tbCfg)
```

Comment:
Let me try that. I see StartTicketPurchase is also called from RPC, where we do not necessarily need to pass in the chain client.